src/ServicePulse.Host/vue/src/components/ExclamationMark.vue

```diff
@@ -1,11 +1,16 @@
-﻿<script setup>
-const props = defineProps({
-  type: String,
-});
+﻿<script setup lang="ts">
+import { LicenseWarningLevel } from "@/composables/LicenseStatus";
+
+const props = withDefaults(
+  defineProps<{
+    type?: LicenseWarningLevel;
```

Comment:
We definitely use this control in other contexts than only licenses. For instance, [we show it when we cannot connect to ServiceControl](https://github.com/Particular/ServicePulse/pull/1706/files#diff-bc13c6fff783e13d1577030cae7a3ae1c35d5e82b1fcab7ed639e3582bbf408cL23).

Reply:
fixed